docs/explanations/architecture.md

```diff
@@ -72,6 +76,49 @@ On a new cluster, the Kubewarden components defined are:
 - the `kubewarden-controller` Deployment
 - a `PolicyServer` Custom Resource named `default`.
 
+<!--TODO: Need to work through this a bit, before progressing to the next
+diagram. Is this OK, how does the text above match with the diagram below. The
+two diagrams are quite different, is anything being lost? Gained? In the
+original, the lines around the planes is dashed. What does that signify? In the
+original there is separation between The control palnes and the K8s API server.
+What are they there for? What are they imparting? There is no further mention
+of them in the text. They are never shown connecting to anything else. If they
+can be removed it may simplify layout as everything becomes more complicated. -->
```

Comment:
I would say we need to rework the diagram in this page. I know what they trying to show. They try to help demonstrate visually what is written. But they are failing on that.  I'm not sure why. Maybe lack of legend? Maybe wrong diagram type? I'm not sure. But I think if they are not helping, we should remove them. 

We can remove the diagrams for now, improve the understanding using only text and re add the diagram again if necessary.